app/src/main/res/drawable-v24/ic_launcher_foreground.xml

```diff
@@ -2,13 +2,13 @@
         xmlns:aapt="http://schemas.android.com/aapt"
         android:width="108dp"
         android:height="108dp"
-        android:viewportHeight="108"
-        android:viewportWidth="108">
+        android:viewportWidth="108"
+        android:viewportHeight="108">
     <path
             android:fillType="evenOdd"
             android:pathData="M32,64C32,64 38.39,52.99 44.13,50.95C51.37,48.37 70.14,49.57 70.14,49.57L108.26,87.69L108,109.01L75.97,107.97L32,64Z"
-            android:strokeColor="#00000000"
-            android:strokeWidth="1">
+            android:strokeWidth="1"
+            android:strokeColor="#00000000">
```

Comment:
Why should you swap them? :)

Reply:
That's because of auto-format. In 9bd07474890377b40df02c4c1d7c4638edc65532 I've run auto-format on the whole project.